client/src/main/java/com/alibaba/nacos/client/config/impl/ServerListManager.java

```diff
@@ -132,28 +132,28 @@ public ServerListManager(List<String> fixed, String namespace) {
         for (String serverAddr : fixed) {
             String[] serverAddrArr = InternetAddressUtil.splitIPPortStr(serverAddr);
             if (serverAddrArr.length == 1) {
-                serverAddrs.add(serverAddrArr[0] + InternetAddressUtil.IP_PORT_SPLITER + ParamUtil.getDefaultServerPort());
+                serverAddrs
```

Comment:
I don't think this change is necessary

Reply:
the modification on ServerListManager  has several purposes:
1.optimize the naming rule of server name  when  the serverlistmanager is using endpoint , port,contentpath ,cluster name which construct the whole address server. 
2.support endpoint query string parameter after the address server url to satisfy more  usage scenarios , for example, this query   string is essential in alibaba internal scene when using address server.
3.support user defined server name to serverlistmanager and client worker , to make our log print more graceful in multi instances  scenarios.